src/Components/ArticleCard.js

```diff
@@ -1,21 +1,22 @@
 import "./ArticleCard.css";
 
-const ArticleCard = (props) => {
-  const date = props.articles.created_at.substring(0, 10);
+const ArticleCard = ({articles, onClick}) => {
+  const date = articles.created_at.substring(0, 10);
 
   return (
-    <div className="articlecard-main__container">
+    <div key={articles.article_id} className="articlecard-main__container" onClick={onClick}>
       <div className="articlecard-image__container">
-        <img src={props.articles.article_img_url} alt="article image" />
+        <img src={articles.article_img_url} alt="article image" />
       </div>
       <div className="articlecard-body__container">
         <div className="articlecard-date-author__container">
-          <div>{props.articles.author}</div>
+          <div>{articles.author}</div>
           <div>{date}</div>
         </div>
         <div className="articlecard-article-description__container">
-          <p className="articlecard-body__text">{props.articles.body}</p>
+          <p className="articlecard-body__text">{articles.body}</p>
         </div>
+
       </div>
     </div>
   );
```

Comment:
There are still quite a lot of <div> tags here. Please try and use more semantic tags